staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```
@@ -443,19 +444,20 @@ func (r *crdHandler) getOrCreateServingInfoFor(crd *apiextensions.CustomResource
 		typer := newUnstructuredObjectTyper(parameterScheme)
 		creator := unstructuredCreator{}
 
-		validator, _, err := apiservervalidation.NewSchemaValidator(crd.Spec.Validation)
+		validationSchema := getCRDSchemaForVersionOrDie(crd, v.Name)
```

same question... why do we want to panic here instead of returning an error?